OmdbSearchMovieDetails/OmdbSearchMovieDetails/Models/SearchAndMovieDetailsModel.swift

```diff
@@ -46,26 +46,28 @@ public struct Search: Codable {
 
 //MARK: - MovieDetails
 @objcMembers public class MovieDetails: NSObject, Codable {
-    public let title: String
-    public let year: String
-    public let rated: String
-    public let released: String
-    public let runtime: String
-    public let genre: String
-    public let director: String
-    public let writer: String
-    public let actors: String
-    public let plot: String
-    public let language: String
-    public let awards: String
-    public let poster: String
-    public let imdbRating: String
-    public let imdbID: String
-    public let boxOffice: String
-    public let type: String
-    public let productionStudio: String
+    public let title: String?
+    public let year: String?
+    public let rated: String?
+    public let released: String?
+    public let runtime: String?
+    public let genre: String?
+    public let director: String?
+    public let writer: String?
+    public let actors: String?
+    public let plot: String?
+    public let language: String?
+    public let awards: String?
+    public let poster: String?
+    public let imdbRating: String?
+    public let imdbID: String?
+    public let boxOffice: String?
+    public let type: String?
+    public let productionStudio: String?
+    public let error: String?
+    public let response: String
     
-    public init(title: String, year: String, rated: String, released: String, runtime: String, genre: String, director: String, writer: String, actors: String, plot: String, language: String, awards: String, poster: String, imdbRating: String, imdbID: String, boxOffice: String, type: String, productionStudio: String) {
+    public init(title: String, year: String, rated: String, released: String, runtime: String, genre: String, director: String, writer: String, actors: String, plot: String, language: String, awards: String, poster: String, imdbRating: String, imdbID: String, boxOffice: String, type: String, productionStudio: String, error: String, response: String) {
```

Comment:
So just a recommendation but it's much more readable when you put each parameter on it's own line, something like this:

```
public init(title: String,
            year: String,
            rated: String,
            released: String,
            etc...) {
 }
```